src/mantine-dates/src/components/TimeInputBase/TimeInputBase.styles.ts

```diff
@@ -8,12 +8,13 @@ export const inputSizes = {
   xl: 29,
 };
 
-interface TimeFieldStyles {
+interface TimeInputBaseStyles {
   size: MantineSize;
+  hasValue: boolean;
 }
 
-export default createStyles((theme, { size }: TimeFieldStyles) => ({
-  timeInput: {
+export default createStyles((theme, { size, hasValue }: TimeInputBaseStyles) => ({
+  input: {
```

Comment:
If it is renamed to just `input`, there will be a collision in Styles API, selector should remain as is, otherwise it will be a breaking change